main.py

```diff
@@ -1060,24 +1214,39 @@ def main():
 
             elif packet.is_type(SMSG_TRADE_COMPLETE):
                 commitMessage=""
-                # The sale_tree is only ammended after a complete trade packet.
+                # we get here which tree it should be through UID
+                if trader_state.item.get == 1:
```

Comment:
Nope, get attribute checks if it's an add operation (0) or a buy/getback (1). This one is ok. Thanks for the items_for_getback, my bad indeed.